frontend/app/components/shared/Filters/FilterAutoComplete/AutocompleteModal.tsx

```
@@ -0,0 +1,190 @@
+    }
+  };
+  const isSelected = (item: { value: string; label: string }) => {
+    return selectedValues.includes(item.value);
+  };
+
+  const applyValues = () => {
+    onApply(selectedValues);
+  };
+
+  const sortedOptions = React.useMemo(() => {
+    if (values[0] && values[0].length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)